src/Eshopworld.Messaging/ServiceBusFluentExtensions.cs

```diff
@@ -99,15 +99,15 @@ public static class TypeExtensions
         public static string GetEntityName(this Type type)
         {
 #if DEBUG
-            var queueName = type.FullName;
+            var queueName = type.Name;
             if (Debugger.IsAttached)
             {
                 queueName += $"-{Environment.UserName.Replace("$", "")}";
             }
 
             return queueName?.ToLower();
 #else
-            return type.FullName?.ToLower();
+            return type.Name?.ToLower();
```

Comment:
ToLowerInvariant. When Name can be null?

Reply:
Honestly I think never. But I was changing name not changing behavior.